src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/ProfessionalUserReqValidatorTest.java

```diff
@@ -9,12 +9,12 @@
     @Test(expected = InvalidRequest.class)
     public void testValidateRequestAllNull() {
         profUserReqValidator
-                .validateRequest(null,null,null, null);
+                .validateRequest(null,null, null);

```

Comment:
Like I said before. It is bad practice to pass in nulls post Java8. This test (and production code) should be refactored to accept Optionals (a Java8 construct designed particularly to handle this).

Reply:
Hi, I did not write this code, I only removed a parameter, but I agree with your comment.